### DISCUSSION TITLE
[Question]hilited_corner_radius 会影响编码区与候选词边距是否符合预期?

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1275] [Q&A]
我感觉这可能是一个 `bug`

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/17060376/234225595-d05dd798-9bf2-4d23-8988-a86d1478120e.png">

**Comment by LEOYoon-Tsaw** [comment]
符合預期

**Reply by pdog18** [reply]
为什么呢? 如果要影响，为什么只影响 top 的边距呢?

**Reply by LEOYoon-Tsaw** [reply]
你調小一點看唄

**Reply by LEOYoon-Tsaw** [reply]
一般都不會設那麼大，如果要考慮設很大的值的情況，可以做更多優化
